data/data/bootstrap/baremetal/files/usr/local/bin/master-bmh-update.sh

```diff
@@ -49,10 +49,13 @@ for node in $(curl -s "${ironic_url}/nodes" | jq -r '.nodes[] | .uuid'); do
      oc annotate --overwrite -n openshift-machine-api baremetalhosts "$name" 'baremetalhost.metal3.io/status'="$HARDWARE_DETAILS" 'baremetalhost.metal3.io/paused-'
```

Comment:
We shouldn't need this HardwareDetails stuff at all any more, because now BMO runs introspection on the control plane hosts.
(The original purpose of this script was to get the introspection data generated by Terraform and forward it to the BMH objects.)

Reply:
Done